.vscode/tasks.json

```diff
@@ -63,7 +63,7 @@
                     "GAZEBO_MODEL_PATH": "${workspaceRoot}/Tools/sitl_gazebo/models"
                 }
             },
-            "command": "gzserver --verbose ${workspaceRoot}/Tools/sitl_gazebo/worlds/iris.world",
+            "command": "gzserver --verbose ${workspaceRoot}/Tools/sitl_gazebo/worlds/iris.world & gzclient",
```

Comment:
I wonder if these should be separate tasks?

Reply:
I was wondering too but didn't know how to do that.

Reply:
I'll take a pass at it when adding a standalone headless option.